### ISSUE TITLE
search sort by popularity

### ISSUE THREAD

**Issue by lifanjiang** [alphasp/pxview#50]
search sort by popularity only one page
Do users must be Premium to show more page？

**Comment by alphasp** [comment]
Yes, to search more than one page, user need to be a pixiv premium user, this is limited by pixiv api.

**Comment by sombresu** [comment]
Is this why sort by popularity only shows 30 pictures?
